blockchain/src/sha256.js

```diff
@@ -0,0 +1,78 @@
+const path = require('path')
+const crypto = require('crypto')
+const ignore = require('ignore')
+const each = require('async-each')
+const sha256File = require('sha256-file')
+
```

Comment:
I think a `const fs = require('fs')` is missing here, the module is used later in the function